python/pyarrow/array.pxi

```diff
@@ -2943,6 +2943,12 @@ cdef class LargeStringArray(Array):
                                   null_count, offset)
 
 
+cdef class StringViewArray(Array):
+    """
+    Concrete class for Arrow arrays of string (or utf8) view data type.
+    """
```

Comment:
Is it worth adding TODOs or linking future GH issues?

Reply:
It's unclear if we need to add any specific method here (at least it's not needed for the current TODO items in the parent issue), so going to leave this as is for now.